.github/workflows/publish-package.yml

```diff
@@ -21,20 +21,21 @@ jobs:
       uses: actions/setup-python@v2
       with:
         python-version: 3.8
-    - name: Install pep517, check-manifest, and twine
+    - name: Install build, check-manifest, and twine
       run: |
         python -m pip install --upgrade pip setuptools wheel
         python -m pip install pep517 --user
```

Comment:
Can't remove `pep517` until an equivalent of `pep517.check` is found. This it the current only hangup for this PR.